docker-compose.yml

```
@@ -722,6 +724,8 @@ services:
       service: microfrontend
     working_dir: '/edx/app/frontend-app-profile'
     container_name: "edx.${COMPOSE_PROJECT_NAME:-devstack}.frontend-app-profile"
+    environment:
+      PARAGON_BRAND_PACKAGE: '@edx/brand-edx.org'
```

As is, this will pull from `latest`, which could introducing breaking changes unintentionally if a new major version is released.

```suggestion
      PARAGON_BRAND_PACKAGE: '@edx/brand-edx.org@~2.0.0'
```

https://github.com/edx/edx-internal/blob/b3bd6adf8407b986349272f33b37922c620d8857/frontends/frontend-app-profile/prod_config.yml#L26